src/components/title-with-description/TitleWithDescription.tsx

```diff
@@ -23,24 +23,15 @@ const TitleWithDescription = ({
   style = styles,
   isDescriptionTooltip = false
 }: TitleWithDescriptionProps) => {
-  const [tooltipVisible, setTooltipVisible] = useState<boolean>(false)
-
-  const handleTooltip = () => setTooltipVisible((prevState) => !prevState)
-
   return (
     <Box sx={style.wrapper}>
       {title && <Typography sx={style.title}>{title}</Typography>}
-      {isDescriptionTooltip ? (
-        <Tooltip open={tooltipVisible} placement='bottom' title={description}>
-          <Typography onClick={handleTooltip} sx={style.description}>
-            {description}
-          </Typography>
-        </Tooltip>
-      ) : (
-        <Typography onClick={handleTooltip} sx={style.description}>
-          {description}
-        </Typography>
-      )}
+      <Tooltip
+        placement='bottom'
+        title={isDescriptionTooltip ? description : ''}
+      >
```

Comment:
Is there a need to use `arrow` prop for tooltip here? How do you think?

Reply:
Good idea! Fixed!